modelci/cli/modelhub.py

```diff
@@ -196,3 +196,39 @@ def detail(model_id: str = typer.Argument(..., help='Model ID')):
     with requests.get(f'{app_settings.api_v1_prefix}/model/{model_id}') as r:
         data = r.json()
         model_detailed_view(MLModel.parse_obj(data))
+
+
+@app.command('convert')
+def convert(
+        way: str = typer.Argument(..., help= 'the type of converting, for example \'keras2onnx\''
+                                             'means convert keras to onnx, available ways: keras2onnx, '
+                                             'tf2onnx, keras2tfs'),
+        model: str = typer.Argument(..., help='path of the model to be converted'
+                                    'such as \'/home/model\''),
+        save: str = typer.Argument(..., help='save path of the converted model'),
+):
+    import modelci.hub.converter.converter as cvt
+    import time
+    if way == 'keras2onnx':
```

Comment:
what about our existing converters?

Reply:
I will set out to them when we come to an agreement on the CLI basic structure.